project.clj

```diff
@@ -1,4 +1,4 @@
-(defproject hikari-cp "2.0.1"
+(defproject hikari-cp "2.0.2-SNAPSHOT"
```

Comment:
Please leave project version as it was :)

Reply:
was following convention of bumping patch version for any new changes beyond the released *.jar

I have reverted `project.clj` back to `2.0.1`